code/game/objects/effects/chem/chemsmoke.dm

```diff
@@ -150,6 +160,24 @@
 		else
 			message_admins("A chemical smoke reaction has taken place in ([whereLink]). No associated key.", 0, 1)
 			log_game("A chemical smoke reaction has taken place in ([where])[contained]. No associated key.")
+	else if (show_touch_log && touch_log_trigger)
```

Comment:
Couldn't the entire variable of `touch_log_trigger` be replaced with a `touched_mobs.len` check? That list only gets populated if there's logs to show.